scout/server/templates/utils.html

```diff
@@ -12,7 +12,7 @@
       	<td>
       	  <small>
       	    <strong> {{ comment.user_name }} </strong>
-      	    on {{ comment.created_at.date() }}
+      	    on {{ comment.created_at.strftime('%d-%m-%Y %H:%M') }}
```

Comment:
But, eh, what kind of date is this?! y-m-d surely!

Reply:
It's the Italian date 😄

Reply:
It would make sense, at least if you do %M:%H %d-%m%-%Y. But let's stick to universal! :)